Makefile.dep

```diff
@@ -981,8 +981,8 @@ ifneq (,$(filter periph_uart_nonblocking,$(USEMODULE)))
   FEATURES_REQUIRED += periph_uart
 endif
 
-# Enable periph_gpio when periph_gpio_irq is enabled
-ifneq (,$(filter periph_gpio_irq,$(USEMODULE)))
+# Add periph_gpio when one or more of its submodules is used
+ifneq (,$(filter periph_gpio_%,$(USEMODULE)))
```

Comment:
Hm, the module `periph_gpio_ext` would then also activate `periph_gpio`, which should not necessarily be the case. Theoretically it would be possible to extend a board with an I2C-GPIO-Extender which does not have the `periph_gpio` feature but has the `periph_i2c` feature.

Reply:
Once the support for external GPIOs is merged, I will rebase and update.

I'm not sure if the API will be implemented by external GPIOs as well, as strict timing requirements do not match well with a serially attached GPIO extender. (Especially with an I2C extender sharing the bus with a notorious clock stretcher.)

Maybe this API should be periph only. Slow bit banging can also be done quite well with xtimer/ztimer and the regular GPIO API.